### ISSUE TITLE
Search landmarks should be unique

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#182]
There are 2 search landmarks on the page:

1. The searchbar which is a `<div>` with role="search"

2. The div which contains the button groups for filtering the search ha role="search"

For this to meet WCAG best practices, all landmarks must have a unique combination of roles, labels or titles. You can read more here:

https://dequeuniversity.com/rules/axe/3.5/landmark-unique
